examples/screenshare/src/index.js

```diff
@@ -1,46 +1,128 @@
   pre.innerHTML = Prism.highlight(snippet, Prism.languages.javascript);
 
+  const logger = Video.Logger.getLogger('twilio-video');
+  logger.setLevel('silent');
+
+  // Connect Local Participant (screen-sharer) to a room
+  const localCreds = await getRoomCredentials();
+  const roomLocal = await Video.connect(localCreds.token, {
+    tracks: []
+  });
+
+  // Connect Remote Participant (screen-viewer) to the room
+  const remoteCreds = await getRoomCredentials();
+  const roomRemote = await Video.connect(remoteCreds.token, {
+    name: roomLocal.name,
+    tracks: []
+  });
+
   // Hide the "Stop Capture Screen" button.
   stopScreenCapture.style.display = 'none';
 
   // The LocalVideoTrack for your screen.
+  let screenStream;
   let screenTrack;
+  let localScreenTrack;
 
   captureScreen.onclick = async function() {
     try {
       // Create and preview your local screen.
-      screenTrack = await createScreenTrack(720, 1280);
-      screenTrack.attach(screenPreview);
+      screenStream = await createScreenStream(720, 1280);
+      screenTrack = await screenStream.getTracks()[0];
+
+      // Publish screen track to room
+      await roomLocal.localParticipant.publishTrack(screenTrack, {
+        priority: 'low',
+      }).then (trackPublication => {
+        localScreenTrack = trackPublication.track;
+        trackPublication.track.attach(screenPreview);
+      });
+
       // Show the "Capture Screen" button after screen capture stops.
-      screenTrack.on('stopped', toggleButtons);
+      localScreenTrack.on('stopped', () => {
+        stopScreenSharing
+      });
+
+      screenTrack.onended = stopScreenSharing;
+
       // Show the "Stop Capture Screen" button.
       toggleButtons();
     } catch (e) {
       alert(e.message);
     }
   };
 
-  stopScreenCapture.onclick = function() {
-    // Stop capturing your screen.
+  // Stop capturing your screen.
+  const stopScreenSharing = () => {
+    roomLocal.localParticipant.unpublishTrack(screenTrack);
     screenTrack.stop();
+    toggleButtons();
+  };
+
+  stopScreenCapture.onclick = stopScreenSharing;
+
+  // Remote Participant handles screen share track
+  if(roomRemote) {
+    roomRemote.on('trackPublished', publication => {
+      trackPublish('publish', publication, remoteScreenPreview);
+    });
+
+    roomRemote.on('trackUnpublished', publication => {
+      trackPublish('unpublish', publication, remoteScreenPreview);
+    });
   }
+
+  // Disconnect from the Room on page unload.
+  window.onbeforeunload = function() {
+    if (roomLocal) {
+      roomLocal.disconnect();
+      roomLocal = null;
+    }
+    if (roomRemote) {
+      roomRemote.disconnect();
+      roomRemote = null;
+    }
+  };
 }());
 
 function toggleButtons() {
   captureScreen.style.display = captureScreen.style.display === 'none' ? '' : 'none';
   stopScreenCapture.style.display = stopScreenCapture.style.display === 'none' ? '' : 'none';
 }
+
+function trackPublish(publishType, publication, view) {
```

Comment:
This is an event handler right? Not sure what is the convention already used in quickstart, but it's better to use something like `onTrackPublished` or `trackPublishHandler`.

Reply:
Good point! Thanks Charlie, it's been updated :)